src/containers/FilterLink.tsx

```diff
@@ -0,0 +1,24 @@
+import { actions, VisibilityFilter } from "../appSlice";
```

Comment:
What's really nice about `@reduxjs/toolkit` is the awesome TypeScript support. Using `dispatch` and `useSelector` feels really good this way.